src/target/renesas.c

```diff
@@ -487,7 +492,8 @@ static bool renesas_rv40_prepare(target_flash_s *f)
 	}
 
 	/* code flash or data flash operation */
-	const bool code_flash = f->start < RENESAS_CF_END;
+	/* Option-Setting flash is CF type as per Table 44.1 of RA4M2 User's Manual */
+	const bool code_flash = f->start < (RENESAS_OF_START + RENESAS_OF_SIZE);
```

Comment:
This looks like it breaks parts with flashes that go over that particular option flash adress, I don't remember if thats possible, but I have to assume I chose the original value for  areason